### PR TITLE
Remove unused plugins

### PR DESCRIPTION
**Pull request by dorukozturk**
Removes julia, swift, vtk and spark-scala plugins.
@girder/developers PTAL.

### CODE REVIEW

**Comment by dorukozturk** [comment]
@zachmullen, @kotfic I resolved the conflicts. PTAL

File: docs/plugins.rst [hunk 1]
```diff
@@ -390,79 +390,3 @@ R
...
-.. _vtkGraphWriter: http://www.vtk.org/doc/nightly/html/classvtkGraphWriter.html
-.. _vtkTree: http://www.vtk.org/doc/nightly/html/classvtkTree.html
-.. _vtkTreeWriter: http://www.vtk.org/doc/nightly/html/classvtkTreeWriter.html
-.. _vtkTable: http://www.vtk.org/doc/nightly/html/classvtkTable.html
-.. _vtkTableWriter: http://www.vtk.org/doc/nightly/html/classvtkTableWriter.html
-.. _vtkPolyData: http://www.vtk.org/doc/nightly/html/classvtkPolyData.html
-.. _vtkPolyDataWriter: http://www.vtk.org/doc/nightly/html/classvtkPolyDataWriter.html
-.. _vtkTree: http://www.vtk.org/doc/nightly/html/classvtkTree.html
-
-Types
```
Comment (zachmullen) [inline docs/plugins.rst]: Was this deletion accidental? The types plugin still exists (to my knowledge).

File: docs/plugins.rst [hunk 2]
```diff
@@ -390,79 +390,3 @@ R
...
-* **Description:** This plugin allows type and format annotations of inputs and outputs
-  to be added to task specs and IO bindings. It can also perform automatic conversion between
-  different formats for known types, as well as validating the correctness of the data formats.
-  These behaviors are enabled with optional boolean arguments to the ``run`` task: ``validate``
-  and ``auto_convert``. Other plugins such as VTK and R add additional types and formats to
-  the typesystem supported by this plugin. The full documentation including a list of supported
-  types and formats can be found in the :ref:`types-and-formats` section.
-
-.. |ra| unicode:: 8594 .. right arrow
-.. |ba| unicode:: 8596 .. bidirectional arrow
```
Comment (zachmullen) [inline docs/plugins.rst]: The above two lines need to remain, they are used in other places in the docs that were not deleted.

File: docs/plugins.rst (hunk 1 above)
Comment (dorukozturk) [inline docs/plugins.rst]: Yes, I will fix it now.

File: docs/plugins.rst (hunk 2 above)
Comment (dorukozturk) [inline docs/plugins.rst]: I fixed this one too.

**Pull request merged**